src/test/test-install-root.c

```diff
@@ -1244,6 +1244,12 @@ int main(int argc, char *argv[]) {
         p = strjoina(root, "/usr/lib/systemd/system-preset/");
         assert_se(mkdir_p(p, 0755) >= 0);
 
+        p = strjoina(root, "/usr/lib/systemd/system/multi-user.target");
+        assert_se(write_string_file(p, "", WRITE_STRING_FILE_CREATE) >= 0);
```

Comment:
Hmm, but an empty file is a mask, i.e. the same as symlinking to /dev/null. Shouldn't there be at least a comment so the file is not empty?

Reply:
Fixed.